src/lambda/setRentalStatus.js

```diff
@@ -0,0 +1,33 @@
+import connectToDatabase from './database/dbConnection';
+import { setRecordStatus } from './database/queries/rentalRecord';
+
+async function handler(event, context) {
+  // eslint-disable-next-line no-param-reassign
+  context.callbackWaitsForEmptyEventLoop = false;
+  if (event.httpMethod !== 'PATCH') {
+    return {
+      statusCode: 405,
+    };
+  }
+
+  try {
+    const request = JSON.parse(event.body);
```

Comment:
You need to validate the mongoId and check if it's valid or not. I found a helper function that you might find useful
```js
const mongoose = require("mongoose");

const { ObjectId } = mongoose.Types;

const isValidMongoObjectId = (id) => {
  if (ObjectId.isValid(id)) {
    if (ObjectId(id).toString() === id) {
      return true;
    }
    return false;
  }
  return false;
};

module.exports = { isValidMongoObjectId };
```